{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -164,12 +164,26 @@
 
 SOCIAL_AUTH_STRATEGY = 'auth_backends.strategies.EdxDjangoStrategy'
 
-# Set these to the correct values for your OAuth2/OpenID Connect provider (e.g., devstack)
-SOCIAL_AUTH_EDX_OIDC_KEY = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_SECRET = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_URL_ROOT = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_LOGOUT_URL = 'replace-me'
-SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY = SOCIAL_AUTH_EDX_OIDC_SECRET
+# Set these to the correct values for your OAuth2 provider (e.g., LMS)
+SOCIAL_AUTH_EDX_OAUTH2_KEY = 'replace-me'
+SOCIAL_AUTH_EDX_OAUTH2_SECRET = 'replace-me'
+SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT = 'replace-me'
+SOCIAL_AUTH_EDX_OAUTH2_LOGOUT_URL = 'replace-me'
+BACKEND_SERVICE_EDX_OAUTH2_KEY = 'replace-me'
+BACKEND_SERVICE_EDX_OAUTH2_SECRET = 'replace-me'
+
+JWT_AUTH = {
+    'JWT_ISSUER': 'http://127.0.0.1:8000/oauth2',
+    'JWT_ALGORITHM': 'HS256',
+    'JWT_VERIFY_EXPIRATION': True,
+    'JWT_PAYLOAD_GET_USERNAME_HANDLER': lambda d: d.get('preferred_username'),
+    'JWT_LEEWAY': 1,
+    'JWT_DECODE_HANDLER': 'edx_rest_framework_extensions.auth.jwt.decoder.jwt_decode_handler',
+    'JWT_PUBLIC_SIGNING_JWK_SET': None,
+    'JWT_AUTH_COOKIE_HEADER_PAYLOAD': 'edx-jwt-cookie-header-payload',
+    'JWT_AUTH_COOKIE_SIGNATURE': 'edx-jwt-cookie-signature',
+    'JWT_AUTH_REFRESH_COOKIE': 'edx-jwt-refresh-cookie',
+}
```

Comment:
Curious: from where did you find these settings?  Not sure if all of them are required.

Reply:
I got them from the Credentials IDA settings here:
https://github.com/edx/credentials/blob/master/credentials/settings/base.py#L361-L379
Happy to remove them if they aren't needed - are they needed?